tests/acceptance/features/apiSpaces/copySpaces.feature

```diff
@@ -451,3 +451,244 @@ Feature: copy file
+    Given using spaces DAV path
+    And user "Brian" has created folder "BRIAN-FOLDER"
+    And user "Brian" has created folder "BRIAN-FOLDER/second-level-folder"
+    And user "Brian" has created folder "BRIAN-FOLDER/second-level-folder/third-level-folder"
+    And user "Brian" has shared folder "BRIAN-FOLDER" with user "Alice"
+    And user "Alice" has accepted share "/BRIAN-FOLDER" offered by user "Brian"
+    And user "Alice" has created folder "Sample-Folder-A"
+    And user "Alice" has created folder "Sample-Folder-A/sample-folder-b"
+    And user "Alice" has created folder "Sample-Folder-A/sample-folder-b/sample-folder-c"
+    When user "Alice" copies folder "Sample-Folder-A/sample-folder-b" from space "Personal" to "/BRIAN-FOLDER/second-level-folder/third-level-folder" inside space "Shares Jail" using the WebDAV API
+    Then the HTTP status code should be "204"
+    And as "Alice" folder "/Sample-Folder-A/sample-folder-b/sample-folder-c" should exist
+    And for user "Alice" folder "BRIAN-FOLDER" of the space "Shares Jail" should contain these files:
+      | /second-level-folder/third-level-folder/sample-folder-c |
+    And the response when user "Alice" gets the info of the last share should include
+      | file_target | /Shares/BRIAN-FOLDER |
+
+
+  Scenario: copy a file into a folder at different level received as a user share
+    Given using spaces DAV path
+    And user "Brian" has created folder "BRIAN-FOLDER"
+    And user "Brian" has created folder "BRIAN-FOLDER/second-level-folder"
+    And user "Brian" has created folder "BRIAN-FOLDER/second-level-folder/third-level-folder"
+    And user "Brian" has shared folder "BRIAN-FOLDER" with user "Alice"
+    And user "Alice" has accepted share "/BRIAN-FOLDER" offered by user "Brian"
+    And user "Alice" has created folder "Sample-Folder-A"
+    And user "Alice" has created folder "Sample-Folder-A/sample-folder-b"
+    And user "Alice" has uploaded file with content "sample file-c" to "Sample-Folder-A/sample-folder-b/textfile-c.txt"
+    When user "Alice" copies folder "Sample-Folder-A/sample-folder-b/textfile-c.txt" from space "Personal" to "/BRIAN-FOLDER/second-level-folder" inside space "Shares Jail" using the WebDAV API
+    Then the HTTP status code should be "204"
+    And for user "Alice" folder "BRIAN-FOLDER" of the space "Shares Jail" should not contain these files:
+      | /second-level-folder/third-level-folder |
+    And as "Alice" file "Sample-Folder-A/sample-folder-b/textfile-c.txt" should exist
+    And for user "Alice" folder "BRIAN-FOLDER" of the space "Shares Jail" should contain these files:
+      | /second-level-folder |
+    And for user "Alice" the content of the file "/BRIAN-FOLDER/second-level-folder" of the space "Shares Jail" should be "sample file-c"
+    And the response when user "Alice" gets the info of the last share should include
+      | file_target | /Shares/BRIAN-FOLDER |
```

Comment:
I find this whole behaviour very weird, for file it get's `overwritten` but for folder it gets added as a sub-folder 🤷🏽‍♀️

Reply:
same applies here as well. its the same behaviour of core  for copy properties.